### DISCUSSION TITLE
Appium 2.0 stable release

### DISCUSSION THREAD

**Discussion by VenkateshSeenivasan** [appium/appium#18564] [Release]
Is there any time frame for the Appium 2.0 stable version release ? Have seen that when I hit the appium.io I am seeing the 2.0 documentation.
Have gone through the discussion about the 2.0 stable release earlier and for most of the queries I can see the reply like "Appium team is working on documentation and once it is done. the stable version will get released".

**Comment by jlipps** [comment]
That's correct. The primary project documentation is done. Still working on updating docs for the core drivers.

Appium 2 is already as stable as it will be when the official release happens. There's no reason not to use it.
